apps/editor/index.d.ts

```diff
@@ -570,8 +570,19 @@ declare namespace toastui {
     public styleToSelectedCells(onStyle: SquireExt, options?: object): void;
   }
 
+  interface MarkdownRange {
+    from: CodeMirror.Position;
+    to: CodeMirror.Position;
+    ollapsed: boolean;
```

Comment:
The value of this property is wrong, `collapsed ` is correct.

Reply:
good point, I will fix this